src/server/ocpp/json/JsonRestWSConnection.ts

```diff
@@ -40,7 +40,7 @@ export default class JsonRestWSConnection extends WSConnection {
 
   public onError(errorEvent: ErrorEvent): void {
     // Log
-    Logging.logError({
+    void Logging.logError({
```

Comment:
Be careful, I sometimes lost logs in prod because of not putting the 'await' keyword.